### ISSUE TITLE
FR: make beds usable

### ISSUE THREAD

**Issue by WulftheNordicRanger** [Andrey2470T/luxury_decor#2]
It would be nice if one could use the beds as actual beds to sleep in and set death spawn/ skip the night and whatnot

**Comment by Andrey2470T** [comment]
Yep, but I will need to write own API as beds.register_bed does nott support nodes of 'mesh' type. I will try to do that.
